docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+version: '3'
+services:
+  junebug:
+    image: praekeltfoundation/junebug:alpine
+    ports:
+      - '80:80'
+      - '9001:9001'
+    links:
+      - rabbitmq
+      - redis
+    environment:
+      - AMQP_HOST=rabbitmq
+      - AMQP_VHOST=/
+      - REDIS_HOST=redis
+    command: jb --channel 'telnet_addr:vumi.transports.telnet.AddressedTelnetServerTransport'
+    restart: always
+
+  public_ussd:
+    build: ../
+    volumes:
+      - ./:/config
+    environment:
+      - AMQP_HOST=rabbitmq
+      - CONFIG_FILE=/config/public.yaml
+    links:
+      - identitystore
+      - hub
+      - sbm
+      - messagesender
+      - redis
+      - rabbitmq
+    restart: always
+  
+  redis:
+    image: redis:alpine
+    restart: always
+
+  rabbitmq:
+    image: rabbitmq:alpine
+    restart: always
+
+  postgres:
+    image: postgres:alpine
+    environment:
+      - POSTGRES_PASSWORD=postgres
+    restart: always
+
+  identitystore:
+    image: praekeltfoundation/seed-identity-store
+    links:
+      - postgres
+    environment:
+      - IDENTITIES_DATABASE=postgres://postgres:postgres@postgres/postgres
```

Comment:
I see all the seed services are connecting to a database called postgres, is this intentional?

Reply:
Kind of. We could change this, but this means that I just need to create a token for one application. Let me take a look to see how difficult it is to have more databases in the postgres docker image